.github/ISSUE_TEMPLATE/---releng-checklist.md

```diff
@@ -32,6 +35,7 @@ title: 'Releng checklist - vX.X'
 
 - [ ] [QA] run screenshot test and upload ([instructions](https://github.com/mozilla-mobile/firefox-tv/wiki/Localization#screenshots))
 - [ ] Upgrade a-c on master
+- [ ] File new Releng checklist with next version and add to top of Backlog
```

Comment:
Is there a reason we want to open the new checklist so far in advance?

Reply:
My thought was that this can a) be batched with some previous checklist items, and b) filing the new bug should be handled by the current releng. Also, getting this in early means we can add it to the sprint during sprint planning.